src/container/container.ts

```diff
@@ -13,6 +13,8 @@ import { getServiceIdentifierAsString } from "../utils/serialization";
 import { ContainerSnapshot } from "./container_snapshot";
 import { Lookup } from "./lookup";
 
+type GetArgs = Omit<interfaces.NextArgs,'contextInterceptor'|'targetType'>
```

Comment:
I prefer not to use Omit

Reply:
Why not ? It is what we are doing by supplying the ommited properties in the _get method

Reply:
It's true, maybe I'm being too picky. I personally prefer to create a model system which creates types from scratch or extending existing types. But its more a preference rather than an argument, a remanent of my OOP background, we don't need to be bounded by that